theorems/T000677.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: T000677
+if:
+  and:
+  - P000185: true
+  - P000137: false
+  
+then:
+  P000206: true
+---
+
+A space $X$ is {P206} if and only if the Kolmogorov quotient of $X$ is {P206}.
+
+But if $X$ is {P185} and not {P137} then its Kolmogorov quotient is non-empty and discrete, so strongly Choquet.
```

Comment:
```suggestion
But if $X$ is {P185} and not {P137} then its Kolmogorov quotient is non-empty and {P52}, so strongly Choquet [(Explore)](https://topology.pi-base.org/spaces?q=Discrete+%2B+not+Empty+%2B+not+Strongly+Choquet).
```

Reply:
It would be better to have an explore link for a claim like this which is not immediately obvious.